frontend/src/components/Bill/Bill.jsx

```diff
@@ -116,26 +117,60 @@ export const Bill = () => {
   } = bill;
 
   return (
-    <div>
-      <div>This is the bill {printNo}&apos;s page</div>
-      <h2>{title}</h2>
-      <p>{summary}</p>
-      <div>
-        <span>
-          {sponsorName}:{sponsorId}
-        </span>
+    <div className="bill-content">
+      <div className="summary">
+        <h2>{title}</h2>
+        <p>{summary}</p>
+        <div className="category">
+          {bill.category} <p>(Sample Category)</p>
+        </div>
+        
+        <p>
+          Sponsored by <span style={{fontWeight:'bold'}}>{sponsorName}</span>
```

Comment:
potential idea: using <b> as a shortcut for bold element

Reply:
sounds good, thank you!